packages/paste-website/src/components/Heading.tsx

```diff
@@ -1,34 +1,78 @@
 import * as React from 'react';
-import {Heading, HeadingProps} from '@twilio-paste/heading';
-import {styled, themeGet} from '@twilio-paste/styling-library';
+import {Box} from '@twilio-paste/box';
+import type {BoxStyleProps} from '@twilio-paste/box';
+import {Text} from '@twilio-paste/text';
+import {Heading} from '@twilio-paste/heading';
+import type {HeadingProps} from '@twilio-paste/heading';
+import {LinkIcon} from '@twilio-paste/icons/esm/LinkIcon';
 import {slugify} from '../utils/RouteUtils';
 
-const StyledAnchorHyperlink = styled.a`
-  text-decoration: none;
-  margin-left: ${themeGet('space.space30')};
-  color: ${themeGet('textColors.colorTextWeak')};
-
-  &:hover {
-    text-decoration: underline;
+const anchoredHeadingSpacing: Partial<
```

Comment:
This may be more readable if we did:
```js
const anchoredHeadingSpacing = {
  heading10: 'space70',
  ....
};

....


<Box alignItems="start" marginBottom={anchoredHeadingSpacing[variant]}>
```

Reply:
That's definitely more readable, but doesn't it lose the typing provided by  
```
Partial<
  {
    [key in HeadingProps['variant']]: Partial<BoxStyleProps>;
  }
>
```
?

Reply:
i talked w/ @SiTaggart about this and all things being equal, I think we'll keep if the original way for now. Thanks Shadi!